src/listeners/linksListener.ts

```diff
@@ -42,7 +42,7 @@ export const linksListener: ListenerInt = {
       }
 
       const linkRegex =
-        /(^|\s+)(([a-z]+:\/\/)?(([a-z0-9-]+\.)+((?!txt|json|css|jsx)[a-z]{3,4}|io|me|gg|to|uk|be|in|eu|ru|us|za)))(:[0-9]{1,5})?(.*\s+|\/|$)/gi;
+        /(^|\s+)(([a-z]+:\/\/)?(([a-z0-9-]+\.)+((?!txt|json|css|jsx|py|js|html|css|scss|ts|rb|cpp|c|h|vue|java|rs|asp|ashx|php|pl|xml|rss)[a-z]{3,4}|io|me|gg|to|uk|be|in|eu|ru|us|za)))(:[0-9]{1,5})?(.*\s+|\/|$)/gi;
```

Comment:
```suggestion
        /(^|\s+)(([a-z]+:\/\/)?(([a-z0-9-]+\.)+((?!txt|json|css|jsx|html|css|scss|cpp|vue|java|asp|ashx|php|xml|rss)[a-z]{3,4}|io|me|gg|to|uk|be|in|eu|ru|us|za)))(:[0-9]{1,5})?(.*\s+|\/|$)/gi;
```
Bot doesn't trigger on 2-letter domains.

Reply:
How do I add regex for two letter domains ?

Reply:
I have fixed it Nicholas, please check.